controllers/yawol-controller/loadbalancerset/loadbalancerset_controller_test.go

```diff
@@ -382,7 +382,29 @@ func TestIsMachineReady(t *testing.T) {
 func TestShouldMachineBeDeleted(t *testing.T) {
 	t.Run("Do not delete if there are no conditions shortly after creation", func(t *testing.T) {
 		machine := &yawolv1beta1.LoadBalancerMachine{
-			ObjectMeta: metav1.ObjectMeta{CreationTimestamp: metav1.Time{Time: time.Now().Add(-9 * time.Minute)}},
+			ObjectMeta: metav1.ObjectMeta{
+				CreationTimestamp: metav1.Time{Time: time.Now().Add(-9 * time.Minute)},
+			},
+			Status: yawolv1beta1.LoadBalancerMachineStatus{
+				Conditions: nil,
+			},
+		}
+		got, _ := shouldMachineBeDeleted(machine)
+		want := false
+
+		if !reflect.DeepEqual(got, want) {
+			t.Errorf("Expected %v got %v", want, got)
+		}
+	})
+
+	t.Run("Do not delete if there are no conditions and the grace period has been adjusted", func(t *testing.T) {
```

Comment:
Can you add another case similar to `Delete if creation is more than 10 minutes old and no conditions` but with a custom creation timeout?